client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -158,50 +157,48 @@ class SiteTools extends Component {
-			const isAtomic = isSiteAutomatedTransfer( state, siteId );
-			const isJetpack = isJetpackSite( state, siteId );
-			const isVip = isVipSite( state, siteId );
-			const isP2 = isSiteWPForTeams( state, siteId );
-			const isP2Hub = isSiteP2Hub( state, siteId );
-			const rewindState = getRewindState( state, siteId );
-			const sitePurchasesLoaded = hasLoadedSitePurchasesFromServer( state );
-
-			const cloneUrl = `/start/clone-site/${ siteSlug }`;
-
-			const copySiteUrl = addQueryArgs( `/setup/copy-site`, {
-				sourceSlug: siteSlug,
-			} );
-
-			const isDevelopmentSite = Boolean( site?.is_a4a_dev_site );
-
-			const showStartSiteTransfer =
-				! isDevelopmentSite && canCurrentUserStartSiteOwnerTransfer( state, siteId );
-
-			return {
-				site,
-				isAtomic,
-				copySiteUrl,
-				siteSlug,
-				purchasesError: getPurchasesError( state ),
-				cloneUrl,
-				showChangeAddress: ! isJetpack && ! isVip && ! isP2,
-				showClone: 'active' === rewindState.state && ! isAtomic,
-				showDeleteContent: isAtomic || ( ! isJetpack && ! isVip && ! isP2Hub ),
-				showDeleteSite: ( ! isJetpack || isAtomic ) && ! isVip && sitePurchasesLoaded,
-				showManageConnection: isJetpack && ! isAtomic,
-				showStartSiteTransfer,
-				siteId,
-				hasCancelablePurchases: hasCancelableSitePurchases( state, siteId ),
-			};
-		},
-		{
-			errorNotice,
-		}
-	),
-] )( localize( withSiteCopy( SiteTools ) ) );
+export default connect(
+	( state ) => {
+		const siteId = getSelectedSiteId( state );
+		const site = getSite( state, siteId );
+		const siteSlug = getSelectedSiteSlug( state );
+		const isAtomic = isSiteAutomatedTransfer( state, siteId );
+		const isJetpack = isJetpackSite( state, siteId );
+		const isVip = isVipSite( state, siteId );
+		const isP2 = isSiteWPForTeams( state, siteId );
+		const isP2Hub = isSiteP2Hub( state, siteId );
+		const rewindState = getRewindState( state, siteId );
+		const sitePurchasesLoaded = hasLoadedSitePurchasesFromServer( state );
+
+		const cloneUrl = `/start/clone-site/${ siteSlug }`;
+
+		const copySiteUrl = addQueryArgs( `/setup/copy-site`, {
+			sourceSlug: siteSlug,
+		} );
+
+		const isDevelopmentSite = Boolean( site?.is_a4a_dev_site );
+
+		const showStartSiteTransfer =
+			! isDevelopmentSite && canCurrentUserStartSiteOwnerTransfer( state, siteId );
+
+		return {
+			site,
+			isAtomic,
+			copySiteUrl,
+			siteSlug,
+			purchasesError: getPurchasesError( state ),
+			cloneUrl,
+			showChangeAddress: ! isJetpack && ! isVip && ! isP2,
+			showClone: 'active' === rewindState.state && ! isAtomic,
+			showDeleteContent: isAtomic || ( ! isJetpack && ! isVip && ! isP2Hub ),
+			showDeleteSite: ( ! isJetpack || isAtomic ) && ! isVip && sitePurchasesLoaded,
+			showManageConnection: isJetpack && ! isAtomic,
+			showStartSiteTransfer,
+			siteId,
+			hasCancelablePurchases: hasCancelableSitePurchases( state, siteId ),
+		};
+	},
+	{
+		errorNotice,
+	}
+)( localize( withSiteCopy( SiteTools ) ) );
```

Comment:
compose doesn't seem to be doing anything here and it was showing up as a typescript error in administration/index.tsx